### DISCUSSION TITLE
Expending on Template

### DISCUSSION THREAD

**Discussion by Nam-Hai** [nuxtrdev/nuxtr-vscode#11] [Ideas]
I love the fact you can setup template for pages and layout, but I find that what I'd need the most would be to have the possibility to setup template for components, heck why not give the possibility to set templates for every type of files you can create with Nuxtr-vscode.

That would I think be a really good quality of life improvement

**Comment by adhamfarrag** [comment]
Hey @Nam-Hai 

Thanks for your kind words.
We're considering all different types of files and different ways to store them globally tool.
